packages/components/src/components/install-guide/record-instructions/Ruby.vue

```diff
@@ -1,69 +1,91 @@
 <template>
   <section>
-    <p class="mb20 bold">
-      When you run your Ruby code with the <tt>appmap</tt> gem in your bundle, AppMap will be
-      enabled for recording. By default, the <tt>appmap</tt> gem is only enabled in the "test" and
-      "development" environments.
+    <p>
+      When you run your Ruby code with the <code class="inline">appmap</code> gem in your bundle,
+      AppMap will be enabled for recording. By default, the <code class="inline">appmap</code> gem
+      is only enabled in the "test" and "development" environments.
     </p>
+    <br />
     <p>
       Before you run your app with AppMap, be sure and stop or disable Spring. You can stop Spring
```

Comment:
Change "be sure and stop" to just "stop"

Reply:
done